test/specs/commonmark/commonmark-spec.js

```diff
@@ -147,22 +107,22 @@ describe('CommonMark 0.28 Indented code blocks', function() {
   var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
 
   cmSpec.forEach(function(spec) {
-    messenger.test(spec, section, ignore);
+    tester.test(spec, section, ignore, testOptions);
   });
 });
 
 describe('CommonMark 0.28 Fenced code blocks', function() {
   var section = 'Fenced code blocks';
 
   // var shouldPassButFails = [];
-  var shouldPassButFails = [93, 95, 96, 97, 101, 102, 106, 108, 111, 112, 113];
+  var shouldPassButFails = [88, 89, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 104, 105, 106, 108, 109, 110, 111, 112, 113, 115];
```

Comment:
Why are more tests failing than before?

Reply:
See `testOptions`. Before I turned `headerIds` off `xhtml` on...this time, I turned `pedantic` off, turned `gfm` off, `headerIds` off,  `xhtml` on, and changed the value for `langPrefix` to match the one from the specification.

Marked doesn't really have a "CommonMark" setting or option. Those were the combination of settings that I thought would us close enough.

Reply:
At least that's my best guess.